From: Michael Biebl <biebl@debian.org>
Date: Mon, 26 Jan 2015 14:58:23 +0100
Subject: Drop rc-local generator

Drop rc-local-generator and instead simply hook up rc-local.service statically
in multi-user.target.wants. Since rc-local.service already as a
ConditionFileIsExecutable=/etc/rc.local, we don't lose any functionality.

Drop halt-local.service service, as halt.local does not exist in Debian.

Bug-Debian: https://bugs.debian.org/776170
---
 Makefile.am | 9 +++++----
 1 file changed, 5 insertions(+), 4 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index ee80874..fcc769b 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -620,12 +620,13 @@ EXTRA_DIST += \
 
 if HAVE_SYSV_COMPAT
 nodist_systemunit_DATA += \
-	units/rc-local.service \
-	units/halt-local.service
+	units/rc-local.service
 
 systemgenerator_PROGRAMS += \
-	systemd-sysv-generator \
-	systemd-rc-local-generator
+	systemd-sysv-generator
+
+MULTI_USER_TARGET_WANTS += \
+	rc-local.service
 endif
 
 EXTRA_DIST += \
